### ISSUE TITLE
Arbitrary values with commas don't work in production builds in Next.js

### ISSUE THREAD

**Issue by wKovacs64** [tailwindlabs/tailwindcss-jit#45]
Hey! First and foremost, thank you for Tailwind and all the content surrounding it. It's amazing and your work is greatly appreciated. And `@tailwindcss/jit` is just... wow. 😍 

---

I took JIT for a spin and noticed at least one class family (`grid-cols-`) is not working in production builds, even though it works in development mode.

For example:

```jsx
function Home() {
  return (
    <div className="px-96">
      <main className="grid grid-cols-[1fr,128px] gap-x-4 items-center p-8">
        <div className="bg-blue-400">Big column</div>
        <div className="bg-gray-400">Small column</div>
      </main>
    </div>
  )
}

export default Home;
```

Development:

```css
.grid-cols-\[1fr\,128px\] {
    grid-template-columns: 1fr 128px;
}
```

![dev](https://user-images.githubusercontent.com/1288694/111255448-ae63af80-85dc-11eb-931e-102e25e0f37f.png)

Production:

(no rules applied for `grid-cols-[1fr,128px]` class)

![prod](https://user-images.githubusercontent.com/1288694/111255461-b3c0fa00-85dc-11eb-91c8-691fe3c4a8e6.png)

Full reproduction with Next.js: https://github.com/wKovacs64/tw-jit-next/tree/issue-45

Any thoughts? Thanks!

**Comment by adamwathan** [comment]
Alright I can reproduce but I have no idea what's happening because it only happens in Next, and if I do the exact same thing with the same plugins outside of Next it works fine. Trying to reach out to maintainer of cssnano-preset-simple who works at Vercel to see if he can help!

**Comment by adamwathan** [comment]
Fixed in v0.1.2 🥳

**Comment by vunamhung** [comment]
I am afraid that you have not counted all the cases.
With prefix like this **_lg_:grid-cols-[1fr,128px]**, it still bug.
I tested with version 0.0.13.
@adamwathan please review this case.
Thank you so much.
